packages/VideoContainer/package.json

```diff
@@ -34,4 +34,4 @@
     ]
   },
   "gitHead": "c7cc1d26b4046b800c727feb1dd283dbd833328c"
-}
+}
```

Comment:
not important so can ignore the comment in any case,  but the JSON files don't have any line breaks.